### PR TITLE
Create glossary component and add it to state page

### LINKED ISSUE #21: Create shell for modal: Glossary
This is a modal that can be opened from the State Report page.

Blocked by #50

### LINKED ISSUE #60: Glossary popup

### PR DESCRIPTION
**Pull request by egiurleo**
This PR creates a component for the glossary modal and adds it to the bottom of the state page!

Screen Shot 2021-02-20 at 7 07 49 PM

Screen Shot 2021-02-20 at 7 07 52 PM

Closes #21.

### CODE REVIEW

File: components/common/Glossary.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    <p>
+      <b>Alternatives to law enforcement responses</b> refers to responses
+      without police, law enforcement or a carceral system (a system that
+      includes prison or jail). Survivor experience and research shows that
+      involving law enforcement in intimate partner violence cases can be more
+      harmful and devastating for survivors. Read{" "}
+      <a
+        href="https://www.freefrom.org/news/2020/6/8/defund-the-police-and-redistribute-resources-to-support-long-term-safety-for-survivors"
+        target="_blank"
+        rel="noopener noreferrer"
```
Comment (DanielKim1) [inline components/common/Glossary.js:16]: :100:

File: components/common/Glossary.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    </p>
+
+    <p>
+      <b>Torts</b> describe what happens when one person injures another. For
+      example, assault, battery, and the intentional infliction of emotional
+      distress are all torts. Tort law allows the person who was injured to sue
+      the person who caused the injury in court to recover some of the costs the
+      injury caused. Some examples of costs that can be covered in a tort case
+      include medical bills, damaged property, mental health costs, and
+      attorneys’ fees.{" "}
```
Comment (DanielKim1) [inline components/common/Glossary.js]: We don't need this space, right?
```suggestion
      attorneys’ fees.
```

**Review by DanielKim1 — APPROVED** [review]
looks good to me!

**Review by sarahgreen — APPROVED** [review]
LGTM! Once Daniel's small suggestion is addressed I think this is good to go.

**Pull request merged**